### PR TITLE
Added missing step

### PR DESCRIPTION
**Pull request by derhallim**
Select settings is required before select sync.

### CODE REVIEW

**Comment by derhallim** [comment]
Related to #2673

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/block-file-types.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ You can prevent users from uploading specific file types when they sync their On
    > 
    > If you have Office 365 operated by 21Vianet (China), [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=850627), then browse to the SharePoint admin center and open the Sharing page.
 
-2. Select **Sync**.
+2. Select **Settings** in the left menu then select **Sync**.
```
Comment (JohanFreelancer9) [inline OneDrive/block-file-types.md:43]:
```suggestion
2. Select **Settings** in the left menu, then select **Sync**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JohanFreelancer9** [comment]
@derhallim - please commit my minor suggested change. Thanks.

**Comment by kaarins** [comment]
Step 1 is "Go to the Settings page of the new SharePoint admin center" and it links directly to the Settings page.

**Pull request closed without merging**